pyproject.toml

```diff
@@ -754,6 +754,7 @@ testing = ["dev", "providers.tests", "tests_common", "tests", "system", "unit",
 "kubernetes-tests/*" = ["D", "TID253", "S101", "TRY002"]
 "helm-tests/*" = ["D", "TID253", "S101", "TRY002"]
 "providers/**/tests/*" = ["D", "TID253", "S101", "TRY002"]
+"performance/tests/*" = ["S101"]
```

Comment:
Why this change? Did you encounter the flake8-bandit’s S101 rule?

Reply:
When I ran `ruff check .` in the repo root it showed a bunch of violations under that path. I figured I might as well fix that too.